components/bl-signature-pad-component/src/actionButtons.js

```diff
@@ -0,0 +1,192 @@
+import SignaturePadButton from './signaturePadButton.js';
+
+export default function ActionButtons(props) {
+  const onClearClick = () => {
+    props.signaturePad.clear();
```

Comment:
a good practice to make a component self-document it to get all props in the top line of the component

```
export default function ActionButtons(props) {
  const { signaturePad, eventHandlers, ... } = props

```